Bottom: 53458a416f4330c1d13afb03e2804060c013f272
Top:    391df273cf6a38233c8bacdfa84c456f50fcd534
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:50:58 +0000

Refresh of buffer-c-declare-variables-bef

---

diff --git a/src/buffer.c b/src/buffer.c
index bec918f..f9d6f5a 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -1449,9 +1449,10 @@ angle is given, the user is prompted for one.
 int
 ActionFreeRotateBuffer(int argc, char **argv, int x, int y)
 {
-  HideCrosshair(false);
   char *angle_s;
 
+  HideCrosshair(false);
+
   if (argc < 1)
     angle_s = gui->prompt_for ("Enter Rotation (degrees, CCW):", "0");
   else
